nntrainer/src/parse_util.cpp

```diff
@@ -21,6 +21,8 @@
  */
 
 #include "parse_util.h"
+#include "nntrainer_error.h"
+#include "nntrainer_log.h"
```

Comment:
If these header files are not in nntrainer/src/ but in another direcotry (refered by -I designated by ninja/meson), don't use "", but use <>.

Reply:
I will change next pr.